### ISSUE TITLE
Buienalarm forecast not updating

### ISSUE THREAD

**Issue by Hendri-88** [Webunity/node-red-contrib-dutch-weather#11]
When inputting {"trigger": true} into the node, the node outputs data of Buienalarm and Buienradar.

The Buienradar output is up to date, but the Buienalarm output is dated the last time the flows where restarted.

After restarting the flows, the first trigger lists current info, but the buienalarm output is stale from that point on.
Please check the screenshot.

Version info:
Node-Red; 2.1.4
node-red-contrib-dutch-weather; 2.0.12
![image](https://user-images.githubusercontent.com/86837491/166159842-16e377bd-ffbf-447d-9fa9-c7e284d1767c.png)

**Comment by gvdhoven** [comment]
Good one, i will see when i have time to take a look at it.

**Comment by rwijbenga** [comment]
Any news on this ? :)

**Comment by gvdhoven** [comment]
No, i have stepped away from this component to be honest. Feel free to 
open a PR and i'll be happy to review it.
